jgiven-maven-plugin/build.gradle

```diff
@@ -21,7 +21,11 @@ task generatePom(type: Copy, dependsOn: copyClasses) {
     }
 }
 
-task generateMavenPlugin(type: CrossPlatformExec, dependsOn: generatePom) {
+Set<Task> findPublishTask(String projectName){
+   rootProject.findProject(projectName).getTasks().findAll {it instanceof PublishToMavenLocal}
+}
+
+task generateMavenPlugin(type: CrossPlatformExec, dependsOn: [generatePom, findPublishTask("jgiven-core"), findPublishTask("jgiven-html5-report")]){
```

Comment:
Can we actually be sure that these two tasks are the only trouble-makers?

Reply:
Well, it's worth a try.

Reply:
Just checked, the setup I wrote indeed put publish tasks on the graph all the time. Rewrote to fix that